### ISSUE TITLE
[BUG] Cannot find module './assets/context/'

### ISSUE THREAD

**Issue by cdaven** [Zettlr/Zettlr#1255]
## Description

I don't know what the effects are, only that there is an error logged. See below.

## Reproducing

Right-click the "horizontal bars icon" for sorting files inside a folder.

## Expected behaviour

Sort the other way? I don't know.

## Platform
 - OS and version: Windows 10
 - Zettlr Version: 1.7.5
 - Screen Resolution: 2560x1440

Installed Zettlr today, via manual download from the website.

## Additional information

```
C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-context.js:302 Error: Cannot find module './assets/context/'
Require stack:
- C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-context.js
- C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-body.js
- C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-renderer.js
- C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\assets\index.htm
    at Module._resolveFilename (internal/modules/cjs/loader.js:797)
    at Function.i._resolveFilename (electron/js2c/renderer_init.js:42)
    at Module._load (internal/modules/cjs/loader.js:690)
    at Function.Module._load (electron/js2c/asar.js:769)
    at Module.require (internal/modules/cjs/loader.js:852)
    at require (internal/modules/cjs/helpers.js:74)
    at ZettlrCon._build (C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-context.js:259)
    at ZettlrCon.popup (C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-context.js:293)
    at C:\Users\ChristianDavén\AppData\Local\Programs\Zettlr\resources\app.asar\renderer\zettlr-body.js:78
```

**Comment by cdaven** [comment]
The sidebar mode is "combined", BTW.

**Comment by nathanlesage** [comment]
Yes, i also encountered the bug and fixed it upstream in the coming 1.8 release!
